doc/userguide/rules/index.rst

```diff
@@ -16,6 +16,7 @@ Suricata Rules
    dns-keywords
    tls-keywords
    ssh-keywords
+   hassh-keywords
```

Comment:
I think hassh should be port of ssh keywords

Reply:
Better to have keywords like: ssh.hassh, ssh.hassh.server, ssh.hassh.string, ssh.hassh.server.string?

Reply:
Yes like ssh.something
Why not `ssh.algorithms` ?

Reply:
Oh hassh seems known already, so ok for `ssh.hassh`